test/smoke_test/smoke_test.py

```diff
@@ -0,0 +1,35 @@
+import os
+import sys
+import torch
+import torchvision
+import torchaudio
+import torch.nn as nn
+import torchaudio.compliance.kaldi  # noqa: F401
+import torchaudio.datasets  # noqa: F401
+import torchaudio.functional  # noqa: F401
+import torchaudio.models  # noqa: F401
+import torchaudio.pipelines  # noqa: F401
+import torchaudio.sox_effects  # noqa: F401
+import torchaudio.transforms  # noqa: F401
+import torchaudio.utils  # noqa: F401
+from torchaudio.io import StreamReader
+import torchvision.datasets as dset
+import torchvision.transforms
+cuda_version_expected = os.environ['CUDA_VER']
+is_cuda_system = cuda_version_expected != "cpu"
+#todo add torch, torchvision and torchaudio tests
+print(torch.__version__)
+print(torchvision.__version__)
+print(torchaudio.__version__)
```

Comment:
minor: perhaps adding a label here would improve clarity of the output, currently it's confusion as to which version corresponds to what:
```
1.13.0.dev20220819
0.14.0.dev20220819
0.13.0.dev20220819
Is torchvision useable? True
11.6
8302
```

e.g.:
```python
print(f"torchaudio: {torchaudio.__version__}")
```